README.md

```diff
@@ -39,10 +39,14 @@ Currently, planet-node produces and uses storage and settings via
 `appsettings.json` and `PN_` prefixed environment variables. if you want to
 change settings, please edit that files or set environment variables.
 
+in bash
 ```
 $ PN_StorePath="/tmp/planet-node" dotnet run --project PlanetNode
 ```
-
+in PowerShell
+```
+$ $Env:PN_StorePath="/tmp/planet-node"; dotnet run --project PlanetNode
+```
```

Comment:
~~~suggestion
```

~~~

Reply:
@zeta6 One remaining here 😅

Reply:
I guess GitHub couldn't detect [a newline](https://github.com/planetarium/planet-node/pull/7/commits/f2408d8f6a64e41de1305c9c3993278a67d07c6c#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R53). don't mind it.